packages/help-center/src/happychat-window-communicator.ts

```diff
@@ -80,13 +82,10 @@ export function useHCWindowCommunicator(
 								{ targetOrigin: event.origin }
 							);
 						}
-						// now clear the store, since we sent everything
```

Comment:
Why 😢

Reply:
I think `resetStore()` is a pretty self explanatory function same thing with `// cleanup`. The comment stating that we are clearing the store seemed redundant. In general I came into this file with no context of HappyChat or how we were talking to HelpCenter and these comments didn't help me with anything I could not deduce by examining.

Reply:
OK I agree with this one.